qaframework-bdd-tests/src/test/java/org/openmrs/contrib/qaframework/automation/AddNewIdentifierSteps.java

```diff
@@ -0,0 +1,76 @@
+package org.openmrs.contrib.qaframework.automation;
+
+import static org.junit.Assert.assertTrue;
+
+import org.openmrs.contrib.qaframework.RunTest;
+import org.openmrs.contrib.qaframework.page.AdministrationPage;
+import org.openmrs.contrib.qaframework.page.PatientFormPage;
+import org.openmrs.contrib.qaframework.page.PatientPage;
+
+import io.cucumber.java.After;
+import io.cucumber.java.Before;
+import io.cucumber.java.en.And;
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
```

Comment:
Leave a space here to make work more readable.

Reply:
this has been added